File: prak3_python/Disk.py
class Disk:

    def __init__(self, type, capacity, price):
    #konstruktor
        self.type = type
        self.capacity = capacity
        self.price = price

    #set
    def setType(self, type):
        self.type = type

    def setCapacity(self, capacity):
        self.capacity = capacity

    def setPrice(self, price):
        self.price = price

    #get
    def getType(self):
        return self.type

    def getCapacity(self):
        return self.capacity

    def getPrice(self):
        return self.price
        

File: prak3_php/Disk.php
<?php

class Disk{

    private $type;
    private $capacity;
    private $price;

    //konstruktor
    function __construct($type, $capacity, $price){
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    //set
    public function setType($type){
        $this->type = $type;
    }

    public function setCapacity($capacity){
        $this->capacity = $capacity;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    //get
    public function getType(){
        return $this->type;
    }

    public function getCapacity(){
        return $this->capacity;
    }

    public function getPrice(){
        return $this->price;
    }

    //destruktor
    function __destruct(){
    }

}


?>